bin/conflicts.js

```diff
@@ -0,0 +1,34 @@
+var PouchDB = require('pouchdb');
+PouchDB.plugin(require('pouchdb-find'));
+var fs = require('fs');
+
+var path = process.argv.slice(2);
+var db = new PouchDB(path[0]);
```

Comment:
Should this be `...path = process.argv[2]...` then just `...new PouchDB(path)...`? Not sure please test.